bosh-stemcell/lib/bosh/stemcell/stemcell_packager.rb

```diff
@@ -71,7 +71,7 @@ def create_tarball(disk_format)
         stemcell_name = ArchiveFilename.new(version, definition, 'bosh-stemcell', disk_format).to_s
         tarball_name = File.join(tarball_path, stemcell_name)
 
-        expected = ['stemcell.MF', 'packages.txt', 'dev_tools_file_list.txt', 'image']
+        expected = ['stemcell.MF', 'packages.txt', 'dev_tools_file_list.txt', 'image', 'sbom.spdx.json', 'sbom.cyclonedx.json']
```

Comment:
According to: https://cyclonedx.org/specification/overview/#recognized-file-patterns
the filename should match `*.cdx.json`